template/desktops/xfce.sh

```diff
@@ -19,6 +19,39 @@ xfconf-query -c xfce4-session -p /startup/ssh-agent/enabled -n -t bool -s false
 xfconf-query -c xfce4-session -p /startup/gpg-agent/enabled -n -t bool -s false
 echo "TIMING: $(date -Iseconds) - Disabled startup services"
 
+# Turn off power saving measures that turn off the display
+xfconf-query \
+    --channel xfce4-power-manager \
+    --property /xfce4-power-manager/dpms-enabled \
```

Comment:
Confirmed this property is `bool` type.

https://gitlab.xfce.org/xfce/xfce4-power-manager/-/blob/xfce4-power-manager-1.6.6/src/xfpm-xfconf.c#L412

```c
    /**
     * XfpmXfconf::dpms-enabled
     **/
    g_object_class_install_property (object_class,
                                     PROP_ENABLE_DPMS,
                                     g_param_spec_boolean (DPMS_ENABLED_CFG,
                                                           NULL, NULL,
                                                           TRUE,
                                                           G_PARAM_READWRITE));
```